notebook/templates/tree.html

```diff
@@ -65,6 +65,7 @@
                   <button class="dropdown-toggle btn btn-default btn-xs" id="new-dropdown-button" data-toggle="dropdown">
                   <span>{% trans %}New{% endtrans %}</span>
                   <span class="caret"></span>
+                  <span class="sr-only">Toggle Dropdown</span>
```

Comment:
Should this have a `{% trans %}` marker for translation?

Reply:
Yes it should and they have been added